package.json

```diff
@@ -64,5 +64,8 @@
   "browserslist": [
     "electron 24.0",
     "defaults"
-  ]
+  ],
+  "dependencies": {
+    "recharts": "^2.8.0"
```

Comment:
We should add this to the `desktop-client` package instead of the root package.

Reply:
Done :)